src/helpers/validateEmail/validateEmail.js

```diff
@@ -0,0 +1,11 @@
+import normalizeInput from '../normalizeInput';
+
+export default email => {
+// eslint-disable-next-line no-useless-escape
```

Comment:
I assume that `validateEmail` should validate email, not the `phoneNumber`? I suggest to split it in to two separate validators and use Facade to combine it